widgets/common/src/filters/FiltersDefinitionForm.tsx

```diff
@@ -1,31 +1,37 @@
 import { useState } from 'react';
 import type { FunctionComponent } from 'react';
 
-import Form from './FormWrapper';
 import RuleRow from './RuleRow';
 import AddRuleButton from './AddRuleButton';
+import type { FilterRule } from './types';
 
-const defaultRow = { attribute: '', operator: '', input: '' };
+interface FilterRuleRow extends FilterRule {
+    id: string;
+}
+
+function getNewRow() {
+    const { uuid } = Stage.Utils;
+    const defaultRow: FilterRuleRow = { id: '', type: 'label', key: '', operator: '', values: [] };
+
+    return { ...defaultRow, id: uuid() };
+}
 
 const FiltersDefinitionForm: FunctionComponent = () => {
-    const [rows, setRows] = useState([defaultRow]);
+    const { UnsafelyTypedForm: Form } = Stage.Basic;
+    const [rows, setRows] = useState([getNewRow()]);
```

Comment:
Optional:

We could pass an initializer function to `useState` to avoid recreating new arrays :slightly_smiling_face: 

```suggestion
    const [rows, setRows] = useState(() => [getNewRow()]);
```

Reply:
Updated.